Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:slim
```

Comment:
Do we need a Dockerfile? Let's discuss.

Reply:
it's easy for people with windows and mac with docker